acquia/acquia_cms
# ACMS-1046: Update composer dependency for acquia_cms_search module

**Motivation**

Fixes #[ACMS-1046](https://backlog.acquia.com/browse/ACMS-1046)

**Proposed changes**

Update composer dependency for acquia_cms_search module

**Alternatives considered**

NA

**Testing steps**


- Run composer require drupal/acquia_cms_search module and make sure it does not download any other acquia_cms modules.

**Merge requirements**
